### DISCUSSION TITLE
Recognize symlinks to license file

### DISCUSSION THREAD

**Discussion by bczhc** [community/community#46602] [Other Feature Feedback, Questions, & Ideas]
### Select Topic Area

Product Feedback

### Body

When license files are symlinked to another license file, on the "About" side panel it shows, like, "Unknown and 6 other licenses found". It only recognizes the non-symlink license file.

**Comment by ghostinhershell** [comment]
Hi @bczhc, we're happy you're here! You are more likely to get a useful response if you are explicit about what your project entails, giving a few more details might help someone give you a nudge in the right direction.

**Comment by bczhc** [comment]
Weird. I can recall and I'm sure after posting this, someone replied me (my emails can prove it), and I also added some supplements about this discussion, but now these comments (mine and of that guy) all disappeared! (I didn't do any "delete" stuff)

Taking this repository https://github.com/bczhc/rime-config, in the license side panel, it shows:
![image](https://github.com/community/community/assets/49330580/e0cce456-841b-4229-aadb-aefa64ed40b1)
All the "Unknown" entries are symlinked to a regular license file, like this:
![image](https://github.com/community/community/assets/49330580/aaff2430-5d55-48d3-90d1-c8fbe8e67d91)
But GitHub won't recognize them and shows "Unknown".

@ghostinhershell Can you take a look at this?

**Comment by pspacek** [comment]
Another example is project https://github.com/isc-projects/bind9 - it has just MPLv2 license + one symlink, and it is displayed like this:

Licenses found

MPL-2.0
LICENSE

Unknown
COPYING

Screenshot: ![image](https://github.com/community/community/assets/11509590/071b2237-74a7-4f9d-a076-ab404a178ea6)

The file `COPYING` is just symlink to `LICENSE`.
